### PR TITLE
[Dubbo-4200] Nacos group should be not resolved from key.

### LINKED ISSUE #4200: no need to override group in  org.apache.dubbo.configcenter.support.nacos.NacosDynamicConfiguration#addListener
- [ ] I have searched the [issues](https://github.com/apache/dubbo/issues) of this repository and believe that this is not a duplicate.
- [ ] I have checked the [FAQ](https://github.com/apache/dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: xxx
* Operating System version: xxx
* Java version: xxx

### Steps to reproduce this issue

```java
public void addListener(String key, String group, ConfigurationListener listener) {
        String[] keyAndGroup = getKeyAndGroup(key);
        if (keyAndGroup[1] != null) {
            group = keyAndGroup[1];
        }
        String finalGroup = group;
        NacosConfigListener nacosConfigListener = watchListenerMap.computeIfAbsent(generateKey(key, group), k -> createTargetListener(key, finalGroup));
        String keyInNacos = rootPath + PROPERTIES_CHAR_SEPERATOR + key;
        nacosConfigListener.addListener(listener);
        try {
            configService.addListener(keyInNacos, group, nacosConfigListener);
            System.out.println("1");
        } catch (NacosException e) {
            logger.error(e.getMessage());
        }
    }
```

i think we should remove the code : group = keyAndGroup[1]   

Pls. provide [GitHub address] to reproduce this issue.

### Expected Result

What do you expected from the above steps？

### Actual Result

What actually happens?

If there is an exception, please attach the exception trace:

```
Just put your stack trace here!
```

### PR DESCRIPTION
**Pull request by ralf0131**
## What is the purpose of the change

Fix #4200 

## Brief changelog

Nacos group should be not resolved from key.

## Verifying this change

Verified locally.

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [x] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [x] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java [hunk 1]
```diff
@@ -209,8 +196,7 @@ public String getConfigs(String key, String group, long timeout) throws IllegalS
     @Override
     public Object getInternalProperty(String key) {
         try {
-            String[] keyAndGroup = getKeyAndGroup(key);
-            return configService.getConfig(keyAndGroup[0], keyAndGroup[1], 5000L);
+            return configService.getConfig(key, null, DEFAULT_TIMEOUT);
```
Comment (uglycow) [inline dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java]: why ignore the  group specified by user

File: dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java [hunk 2]
```diff
@@ -93,24 +97,14 @@ private ConfigService buildConfigService(URL url) {
         return configService;
     }
 
-    public void publishNacosConfig(String key, String value) {
+    public void publishNacosConfig(String key, String group, String value) {
```
Comment (cvictory) [inline dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java]: I don't find the usage of this method.

Comment (ralf0131) [reply dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java]: It is in test code, NacosDynamicConfigurationTest

Comment (ralf0131) [reply dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java]: It is in test code, see `NacosDynamicConfigurationTest`

File: dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java (hunk 1 above)
Comment (ralf0131) [reply dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java]: The null value should be change to "dubbo" instead.

File: dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java [hunk 3]
```diff
@@ -157,48 +134,68 @@ private void putPropertyIfAbsent(URL url, Properties properties, String property
      * @return
      */
     private NacosConfigListener createTargetListener(String key, String group) {
+        String normalizedKey = normalizedKey(key);
```
Comment (chickenlj) [inline dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java]: duplicate call of `normalizedKey(key)`, the caller of createTargetListener() has already called the same method, so the key passed in is a normalized one.

Comment (ralf0131) [reply dubbo-configcenter/dubbo-configcenter-nacos/src/main/java/org/apache/dubbo/configcenter/support/nacos/NacosDynamicConfiguration.java]: Fixed.

**Pull request merged**